proknow-sdk/Scorecard/ScorecardTemplates.cs

```diff
@@ -51,6 +53,13 @@ internal ScorecardTemplates(ProKnowApi proKnow)
                 throw new ArgumentNullException("customMetrics");
             }
 
+            string workspaceID=null;
```

Comment:
Space for all assignments 
```suggestion
            string workspaceID = null;
```

Reply:
Updated